astropy/coordinates/tests/test_regression.py

```diff
@@ -690,3 +690,17 @@ def test_regression_10291():
     venus = get_body('venus', t)
     assert_quantity_allclose(venus.separation(sun),
                              554.427*u.arcsecond, atol=0.001*u.arcsecond)
+
+
+def test_regression_11761():
+    # The output of SkyCoord.apply_space_motion() should have the same
+    # differential type as the SkyCoord.
+    sc1 = SkyCoord(ra=1*u.deg, dec=1*u.deg, distance=1*u.pc,
```

Comment:
We don't test numbers here, but it may be good to do so. E.g., picking `dec=60*u.deg`, there is a factor 2 difference between `pm_ra` and `pm_ra_cosdec`.